keps/prod-readiness/sig-node/3317.yaml

```diff
@@ -0,0 +1,7 @@
+# The KEP must have an approver from the
+# "prod-readiness-approvers" group 
+# of http://git.k8s.io/enhancements/OWNERS_ALIASES
+kep-number: 3317
+alpha:
+  # I picked up from the list as doc says, also asked in slack
```

Comment:
nit: I think you can remove this comment for now since @wojtek-t reviewed already

Reply:
+1 - let's remove it

Reply:
Thanks, fixed now!